### PR TITLE
Description height & scroll fixed

### LINKED ISSUE #160: description container should have max height and allow overflow content to be scrollable
This should be done in tailwind. Here's the css just so you know how its done with without tailwind:

```css
.Description {
  max-height: 300px;
  overflow: scroll;
}
```

### PR DESCRIPTION
**Pull request by akshara-sun**
closes #160

### CODE REVIEW

**Review by amilajack — CHANGES_REQUESTED** [review]
Bug

Screen Shot 2020-11-12 at 9 31 22 PM

**Review by amilajack — CHANGES_REQUESTED** [review]
Bug

Screen Shot 2020-11-12 at 9 43 47 PM

**Review by amilajack — CHANGES_REQUESTED** [review]
Bug

Screen Shot 2020-11-12 at 10 20 08 PM

File: src/Sound.js [hunk 1]
```diff
@@ -174,10 +174,12 @@ export default function Sound(props) {
           {/* Description */}
           <div className="pb-16">
             <h1 className="text-left text-3xl pt-24 py-4">Description</h1>
-            <p
-              className="bg-secondary-static text-primary-static border border-primary-static text-lg p-10"
-              dangerouslySetInnerHTML={sanitizedHTMLFromFreesound}
-            />
+            <div className="Description">
+              <p
+                className="bg-secondary-static text-primary-static overflow-scroll border border-primary-static text-lg p-10"
```
Comment (amilajack) [inline src/Sound.js:179]: Move all these classes to parent to fix this

Comment (akshara-sun) [reply src/Sound.js]: @amilajack you said that the max height should be double the current height, which is set to 300px, so I am going to push another commit where the max height is set to 600px. Hope this is what you're looking for.

Comment (akshara-sun) [reply src/Sound.js]: Fixed

**Review by amilajack — APPROVED** [review]

**Pull request merged**